src/OpenTelemetry.Instrumentation.SqlClient/SqlClientTraceInstrumentationOptions.cs

```diff
@@ -132,7 +132,7 @@ public class SqlClientTraceInstrumentationOptions
     /// cref="Activity"/>.</item>
     /// </list>
     /// </remarks>
-    public Action<Activity, string, object> Enrich { get; set; }
+    public Action<Activity, string, object>? Enrich { get; set; }
```

Comment:
Just double checking if object (SqlCommand) is always non null here.

Reply:
I'm not sure on the requirements, but reviewing how Enrich is invoked:

`SqlClientDiagnosticListener` first checks if command is null before invoking.


https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/c34b5cd08ca9b81f04a85a0503984d7e479c4e67/src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientDiagnosticListener.cs#L67-L72


https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/c34b5cd08ca9b81f04a85a0503984d7e479c4e67/src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientDiagnosticListener.cs#L133

Reply:
Thanks for confirmation